freshmaker/lightblue.py

```diff
@@ -1218,6 +1218,8 @@ def find_parent_brew_build_nvr_from_child(self, child_image):
         :rtype: str
 
         """
+        if not child_image.get("parent_build_id"):
+            return None
```

Comment:
In case this image has not been resolved, it will return before resolving which is unexpected. So we need to do this after resolving https://github.com/redhat-exd-rebuilds/freshmaker/blob/f9bac30955792b1d8a6e0c43447a11afbe970334/freshmaker/lightblue.py#L1226

Please add a comment there to explain why we need to check the `parent_build_id`.